### ISSUE TITLE
Display Choice cuts off text with variable width font

### ISSUE THREAD

**Issue by pcmdx** [chrismaltby/gb-studio#1637]
**Describe the bug**
For certain strings, the Display Choice event cuts off text even though there's room for the characters.

**To Reproduce**
1. Create a Display choice even
2. Type "Seduction book (Atn)"
3. View the preview of the choice, which shows it fits
4. Run game, which shows missing characters

**Expected behavior**
Show all characters up to the end of the popup.

**Screenshots**
![image](https://github.com/user-attachments/assets/9e99c4ff-afa3-45d9-a189-6cbcaaff74c5)

**Platform (please complete the following information):**
 - OS: Linux Mint
 - App version: 4.1.3

**Additional context**
If I set the project to mono spaced, it cuts the text off at the same character, making me think the Display Choice event is using the wrong code to determine the cut off. 

![image](https://github.com/user-attachments/assets/93c4be87-1be4-453f-8928-5552665f4f63)

**Comment by chrismaltby** [comment]
Yeah you were right this was left over cropping from a time when only mono space fonts were supported so cropping was handled automatically. These days most of that auto cropping has been removed and it's up to you to make sure you don't write too much text, I'd just missed removing that that one.

I've fixed here:
https://github.com/chrismaltby/gb-studio/commit/ae071b4d89d4c33971a7b206cc8822cc10ca01d5
which will be included in the next release.

Another option you have rather than updating is to use the "Display Menu" event with the following options which should end up being equivalent but without the cropping
<img width="470" alt="Screenshot 2024-11-09 at 11 12 54" src="https://github.com/user-attachments/assets/3be90a98-9dd2-4285-a855-9cca30fecd95">

**Comment by pcmdx** [comment]
Thanks Chris. Appreciate the fix AND the workaround. Your work has helped me release my Death Row game and I'm working on the sequel!
